src/commons/print-conversation.js

```diff
@@ -0,0 +1,329 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { forEach, reduce, map, filter, isEmpty } from 'lodash';
+import moment from 'moment';
+
+const _CI_REGEX = /^<(.*)>$/;
+const _CI_SRC_REGEX = /^cid:(.*)$/;
+
```

Comment:
Sorry I forgot to reuse the code ,the plainTextToHTML & findAttachments is already available in src/commons/mail-message-renderer.jsx ,you can export them from there and directly import it here to reduce the code duplicacy